independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/instance/PriorityTest.java

```diff
@@ -35,8 +35,7 @@ interface Converter {
 
     }
 
-    @jakarta.annotation.Priority(5) // this priority takes precedence
-    @Priority(50)
+    @Priority(5)
```

Comment:
This tested conflict of both annotations - not needed anymore

Reply:
We could remove of the beans but we still want to keep what the test asserts (ordering of instances while iterating over them) so I've just kept this as-is.